app/root.tsx

```diff
@@ -84,6 +84,7 @@ function Document({ children, title }: PropsWithChildren<{ title?: string }>) {
 
         <Meta />
         {title ? <title>{title}</title> : null}
+        <style data-fullcalendar />
```

Comment:
Not sure what this style tag is? Remix has its own way of adding styles, can we not use that?

Reply:
This can be removed. Thanks for pointing out!